index.html

```diff
@@ -471,13 +471,16 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         through a transformation function (<b>iteratee</b>). The <tt>iteratee</tt>
         is passed three arguments: the <tt>value</tt>, then the <tt>index</tt>
         (or <tt>key</tt>) of the iteration, and finally a reference to the entire
-        <tt>list</tt>.
+        <tt>list</tt>. Selective <b>Underscore</b> functions marked as <tt>guarded</tt>
+        can be used as an <tt>iteratee</tt> to <b>_.map</b>.
```

Comment:
Yeah, I see your point. But I don't think the example you are referencing is the type of behavior I would expect. The fact that guarded methods can be passed into functions like `_.filter` is more of a side effect rather than something which was explicitly implemented. Hence, why the annotated source explicitly states guarded functions work with `_.map`.